modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/view.jsp

```diff
@@ -43,6 +43,8 @@ ContentDashboardAdminManagementToolbarDisplayContext contentDashboardAdminManage
 						>
 							<span class="lfr-portal-tooltip" title="<%= LanguageUtil.get(request, "flip-axes") %>">
 								<form action="<%= contentDashboardAdminDisplayContext.getSwapConfigurationURL() %>" method="post">
+									<aui:input name="redirect" type="hidden" value="<%= contentDashboardAdminDisplayContext.getRedirect() %>" />
```

Comment:
Do we need to add the contentDashboardAdminDisplayContext.getRedirect() here? 
 
If there is no any logic linked with contentDashboardAdminDisplayContext.getRedirect(), why we just not do
	<aui:input name="redirect" type="hidden" value="<%= currentURL %>" />

Reply:
Ohh, that's true! much much easier, Changing...